nullaway/src/main/java/com/uber/nullaway/fixserialization/FixSerializationConfig.java

```diff
@@ -136,8 +136,8 @@ private SerializationAdapter initializeAdapter(int version) {
     switch (version) {
       case 1:
         return new SerializationV1Adapter();
-      case 2:
-        return new SerializationV2Adapter();
+      case 3:
```

Comment:
Let's add a explicit case for v2 that throws a `RuntimeException(...)` noting that v2 is skipped and was used for an alpha version of the auto-annotator tool. Recommend the user to use `v3` instead.

Reply:
Sure, please find it here 56f58befa746f1cfaf947ec3d04676ce23a7fc8d